tools/src/test/python/dlpx/virtualization/_internal/test_schema_validator.py

```diff
@@ -21,7 +21,7 @@ def test_bad_meta_schema(schema_file, tmpdir, schema_filename):
             validator.validate()
 
         message = err_info.value.message
-        assert ('Failed to load schemas because {!r} is not a valid json file.'
+        assert ('Failed to load schemas because \'{}\' is not a valid json file.'
                 ' Error: Extra data: line 2 column 1 - line 2 column 9'
                 ' (char 19 - 27)'.format(schema_file)) in message
```

Comment:
per our style guidelines, if your string will have single quotes you'll want to switch this to use double quotes rather than escape the single quotes, so:
```
assert ("Failed to load schemas because '{}' is not a valid json file."
             " Error: Extra data: line 2 column 1 - line 2 column 9"
             " (char 19 - 27)".format(schema_file)) in message
```